parsers/CH.py

```diff
@@ -9,32 +10,60 @@
 from parsers.lib.config import refetch_frequency
 
 
-def get_solar_capacity_at(date: datetime) -> float:
+def get_solar_capacity_at(target_datetime: datetime) -> float:
+    """Returns the solar capacity (in MW) at a given time.
+
+    References:
+        https://www.uvek-gis.admin.ch/BFE/storymaps/EE_Elektrizitaetsproduktionsanlagen/?lang=en
+        https://www.uvek-gis.admin.ch/BFE/storymaps/EE_Elektrizitaetsproduktionsanlagen/data/solar.csv
+    """
+
     # Prepare historical records
-    # Source https://www.uvek-gis.admin.ch/BFE/storymaps/EE_Elektrizitaetsproduktionsanlagen/?lang=en
-    historical_capacities = pd.DataFrame.from_records(
-        [
-            ("2015-01-01", 1393),
-            ("2016-01-01", 1646),
-            ("2017-01-01", 1859),
-            ("2018-01-01", 2090),
-            ("2019-01-01", 2375),
-            ("2020-01-01", 2795),
-            ("2021-01-01", 3314),
-            ("2022-01-01", 3904),
-        ],
-        columns=["datetime", "capacity.solar"],
-    ).set_index("datetime")
-    historical_capacities.index = pd.DatetimeIndex(
-        historical_capacities.index, tz="UTC"
+    historical_data = """
+        Power_sum,Year,Plant_count
+        6.213,2000,404
+        7.668,2001,476
+        9.09,2002,536
+        10.554,2003,644
+        12.353,2004,731
+        16.994,2005,850
+        19.241,2006,1044
+        26.007,2007,1514
+        42.472,2008,2725
+        80.181,2009,4627
+        123.568,2010,6489
+        250.203,2011,11138
+        500.19,2012,18076
+        806.067,2013,25407
+        995.055,2014,32790
```

Comment:
While great that you found all this data we don't need anything before 2015-01-01 as that is the absolute earliest we have some confidence in the data we are collecting and flowtracing. (2015 is when the ENTSO-E transparency platform launched).